2.4/alpine/Dockerfile

```diff
@@ -0,0 +1,64 @@
+FROM alpine:3.3
+
+# add our user and group first to make sure their IDs get assigned consistently, regardless of whatever dependencies get added
+RUN adduser -D www-data
```

Comment:
Do you think we ought to hard-code this with UID/GID 82 like we did in PHP (https://github.com/docker-library/php/blob/27a370a6b7d254dd8726955a5eabd92a7562f32d/Dockerfile-alpine.template#L19-L26) to ensure we don't cause any conflicts with Alpine packages down the road?